### ISSUE TITLE
_repr_json_ favored over widget output in notebooks with ipywidgets v8.0.1

### ISSUE THREAD

**Issue by lukasbindreiter** [jupyter-widgets/ipywidgets#3576]
## Description

I noticed that with a recent upgrade to `ipywidgets==8.0.1` the behaviour of widgets that also have a `_repr_json_` method changed.

## Reproduce

```python
import ipywidgets

class MyFloatProgress(ipywidgets.FloatProgress):
    def _repr_json_(self):
        return {"value": self.value}

MyFloatProgress(30)
```

![image](https://user-images.githubusercontent.com/21131639/187166811-41fab701-05d8-45a4-bbbd-8d8735546eca.png)

## Expected behavior

![image](https://user-images.githubusercontent.com/21131639/187166868-b0e9d234-ff40-474c-a4a8-9ddf10b9bd74.png)

## Context

This may seem like a pretty niche issue, but it is actually the underlying cause why `tqdm.notebook` doesn't work anymore currently with `ipywidgets==8.0.1`: 

See:
https://github.com/tqdm/tqdm/issues/1359

![image](https://user-images.githubusercontent.com/21131639/187167445-754b282d-ebf8-41d1-bade-82a0c5a656d4.png)

### If using JupyterLab
- JupyterLab version:
3.4.5

<details><summary>Installed Labextensions</summary>
<pre>
JupyterLab v3.4.5
/home/grasp/.venv/share/jupyter/labextensions
        dask-labextension v5.3.0 enabled OK (python, dask_labextension)
        jupyter-matplotlib v0.11.2 enabled OK
        jupyterlab-link-share v0.2.4 enabled OK (python, jupyterlab-link-share)
        jupyterlab-system-monitor v0.8.0 enabled OK (python, jupyterlab-system-monitor)
        jupyterlab-topbar-extension v0.6.1 enabled OK (python, jupyterlab-topbar)
        jupyterlab_pygments v0.2.2 enabled OK (python, jupyterlab_pygments)
        @hadim/jupyter-archive v3.3.1 enabled OK (python, jupyter-archive)
        @jupyter-server/resource-usage v0.6.1 enabled OK (python, jupyter-resource-usage)
        @jupyter-widgets/jupyterlab-manager v5.0.2 enabled OK (python, jupyterlab_widgets)
        @jupyterlab/server-proxy v3.2.1 enabled OK
        @ryantam626/jupyterlab_code_formatter v1.5.3 enabled OK (python, jupyterlab-code-formatter)

Other labextensions (built into JupyterLab)
   app dir: /home/grasp/.venv/share/jupyter/lab
        @jupyterlab/geojson-extension v3.1.2 enabled OK
        @jupyterlab/katex-extension v3.2.2 enabled OK
        @jupyterlab/vega3-extension v3.1.3 enabled OK

Disabled extensions:
    @jupyterlab/mathjax2-extension:plugin
</pre>
</details>

**Comment by jasongrout** [comment]
Thanks for reporting this. We already register the placeholder widget rendermime factory with rank 0, so that should beat out the json renderer in priority. My guess is that we should also add the same rank 0 argument to the notebook-specific rendermime at https://github.com/jupyter-widgets/ipywidgets/blob/e722a0d30637d598d20305f2075d232678518aa8/python/jupyterlab_widgets/src/plugin.ts#L127

**Comment by jasongrout** [comment]
> So we can move to a negative rank in our code, and/or ask upstream jlab why the json renderer is rank 0 instead of, say, rank 50 if it needs to beat out some other default renderers.

I moved our rank to -10 in https://github.com/jupyter-widgets/ipywidgets/pull/3579

**Comment by jasongrout** [comment]
This should be fixed in ipywidgets 8.0.2, which was just published.

**Comment by lukasbindreiter** [comment]
Awesome, thanks :smile:
